docs/modules/ROOT/pages/authentication-and-authorization/configuration.adoc

```diff
@@ -20,7 +20,7 @@ const server = new ApolloServer({
 const { url } = await startStandaloneServer(server, {
     listen: { port: 4000 },
     context: async ({ req }) => ({
-        token: req.headers.Authorization,
+        token: req.headers.authorization,  // use headers.authorization even if your header has "Authorization" as key for the token
```

Comment:
Could you instead of adding the information as a comment move it to just after the code block?
Similar to:
```
    }),
});
----
Note the `context` function: Specify the `authorization` in the `req.headers` as all lowercase despite the HTTP header's initial capital letter.

Optionally, if a custom decoding mechanism is required, that same header can be decoded and the resulting JWT payload put into the `jwt` field of the context.
```
```suggestion
        token: req.headers.authorization,
```